src/components/game/game.tsx

```diff
@@ -32,14 +32,21 @@ export default function Game() {
     function nextTurn() {
         const newBoard = getNextBoard();
         const hasAnyChanges = getHasAnyChanges(board, newBoard);
+        const isEmptyBoard = getIsEmptyBoard(newBoard);
+
         if (hasAnyChanges) {
             setBoard(newBoard);
         }
+        else if(isEmptyBoard) {
+            setIsRunning(false);
+            setMessage(`GG, you did it! It took ${turnCounter} iterations. Next time it won't be as easy!`);
+        }
         else {
```

Comment:
`
else {
  const isEmptyBoard = getIsEmptyBoard(newBoard);
  setIsRunning(false);
  const selectedMessage = isEmptyBoard ? 'GG, you did it! It took ${turnCounter} iterations. Next time it won't be as easy!' : 'You failed to stop the virus from spreading. It took ${turnCounter} iterations.'
  setMessage(selectedMessage );
}
`